src/modules/security/auth/auth.module.ts

```diff
@@ -0,0 +1,32 @@
+import { Module } from '@nestjs/common';
+import { Bcrypt } from '../bcrypt/bcrypt';
+import { UserModule } from '../../entities/user/user.module';
+import { AuthService } from './services/auth.service';
+import { PassportModule } from '@nestjs/passport';
+import { LocalStrategy } from './strategies/local.strategy';
+import { AuthController } from './controllers/auth.controller';
+import { JwtModule } from '@nestjs/jwt';
+import { JwtStrategy } from './strategies/jwt.strategy';
+import { ConfigService } from '@nestjs/config';
+
+@Module({
+  imports: [
+    UserModule,
+    PassportModule,
+    JwtModule.registerAsync({
+      useFactory: (config: ConfigService) => {
+        return {
+          secret: config.get<string>('JWT_SECRET'),
+          signOptions: {
+            expiresIn: '1d',
```

Comment:
Idea with JWT is that they last as short at possible, it should be <1h and then app uses refresh tokens endpoint to receive new one that it rotates. I usually set around 20min

Reply:
Okay, I'll put the suggested value.